src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java

```diff
@@ -433,14 +442,21 @@ public boolean checkSlotOnPageLoaded(String slotName) {
     WebElement slot;
     changeImplicitWait(250, TimeUnit.MILLISECONDS);
     try {
+      String slotSelector = AdsContent.getSlotSelector(slotName);
+
       if (slotName.equals(AdsContent.FLOATING_MEDREC)) {
-        triggerAdSlot(AdsContent.FLOATING_MEDREC);
+        triggerAdSlot(AdsContent.FLOATING_MEDREC).
+            wait.
+            forElementPresent(By.cssSelector(slotSelector));
```

Comment:
this \n looks strange here. If we want to split into multiple lines let's put `.` at the beginning of the line